doc/rados/configuration/mon-lookup-dns.rst

```diff
@@ -42,10 +42,15 @@ With those records now existing we can create the SRV TCP records with the name
 
 ::
 
-    _ceph-mon._tcp.example.com. 60 IN SRV 10 60 6789 mon1.example.com.
-    _ceph-mon._tcp.example.com. 60 IN SRV 10 60 6789 mon2.example.com.
-    _ceph-mon._tcp.example.com. 60 IN SRV 10 60 6789 mon3.example.com.
+    _ceph-mon._tcp.example.com. 60 IN SRV 10 20 6789 mon1.example.com.
+    _ceph-mon._tcp.example.com. 60 IN SRV 10 30 6789 mon2.example.com.
+    _ceph-mon._tcp.example.com. 60 IN SRV 20 50 6789 mon3.example.com.
 
-In this case the Monitors are running on port *6789*, and their priority and weight are all *10* and *60* respectively.
+Now all Monitors are running on port *6789*, with priorities 10, 10, 20 and weights 20, 30, 50 respectively.
 
-The current implementation in clients and daemons will *only* respect the priority set in SRV records, and they will only connect to the monitors with lowest-numbered priority. The targets with the same priority will be selected at random.
+If specified, priorities and weights will be treated in the following order:
+1. priorities always take precedence over weights, the lower value the more preferred
```

Comment:
https://github.com/ceph/ceph/pull/26940#discussion_r266862680

use 
```rst
#. first
#. second
``` 
for ordered items